x/meshsecurity/keeper/genesis.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity/types"
+)
+
+func (k Keeper) InitGenesis(ctx sdk.Context, data types.GenesisState) {
+	if err := k.SetParams(ctx, data.Params); err != nil {
+		panic(err)
+	}
+}
+
+func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
+	params := k.GetParams(ctx)
+	return types.NewGenesisState(params)
+}
```

Comment:
It would be nice to have some unit test for init/export to have this covered. There are more fields missing in the genesis and when there is a test already, we will get better contributions